### ISSUE TITLE
Hyperparameter tuning not done while evaluating  beta vae metric

### ISSUE THREAD

**Issue by kadarsh22** [google-research/disentanglement_lib#28]
The main limitation of beta vae metric is that it is sensitive to hyper parameters . This is mentioned in disentangling by factorizing. By not tuning the learning rate of Logistic  regressior we need not get the true beta vae metric
